src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -94,6 +95,24 @@ public class EnterpriseStartMojo extends AbstractEnterprisePluginMojo {
   @Parameter(property = "gatling.enterprise.simulationSystemPropertiesString")
   private String simulationSystemPropertiesString;
 
+  /**
+   * Provides additional environment variables when starting a simulation, in addition to the ones
+   * which may already be configured for that simulation (see
+   * https://gatling.io/docs/enterprise/cloud/reference/user/simulations/#step-3-injector-parameters).
+   */
+  @Parameter(property = "gatling.enterprise.simulationEnvironmentVariables")
```

Comment:
1. This property isn't actually usable, we should remove the `property` param here as well as for `simulationSystemProperties`.
2. We give a higher priority to `simulationEnvironmentVariablesString` (mainly useful to inject the values with a system property) over `simulationEnvironmentVariables` (which can only be configured in the POM). But in the standard situation where we don't need to define two parameters, Maven gives a higher priority to the configuration in the POM over the system property (I haven't seen it [in the documentation](https://maven.apache.org/guides/mini/guide-configuring-plugins.html), but it's what happens when I test it e.g. with `simulationClass`). I would do the opposite.
3. I think we should more clearly communicate the intent: `simulationEnvironmentVariables` is easier to configure in the POM, `simulationEnvironmentVariablesString` is provided mainly to allow setting with a system property. Maybe something like:
  ```java
  /**
   * Provides additional environment variables when starting a simulation, in addition to the ones
   * which may already be configured for that simulation (see
   * https://gatling.io/docs/enterprise/cloud/reference/user/simulations/#step-3-injector-parameters).
   *
   * <p>To provide environment variables on the command line, you can use {@link
   * #simulationEnvironmentVariablesString} instead, e.g. {@code
   * -Dgatling.enterprise.simulationEnvironmentVariablesString="key1=value1,key2=value2"}.
   */
  @Parameter private Map<String, String> simulationEnvironmentVariables;

  /**
   * Alternative to {@link #simulationEnvironmentVariables} which takes a specially formatted {@code
   * String} instead of a key-value map. The expected format is {@code key1=value1,key2=value2}.
   * Useful to allow configuration on the command line, rather than in the POM, with the associated
   * property.
   */
  @Parameter(property = "gatling.enterprise.simulationEnvironmentVariablesString")
  private String simulationEnvironmentVariablesString;
  ```